Builds failing due to Eclipse repo changes?

I think we encounter this kind of build failures from time to time when there are changes to the Eclipse repos. If I remember correctly, we need to rebuild the target platform by updating the `.tpd` files.

```
29084 [ERROR] Failed to resolve target definition /home/travis/build/GoogleCloudPlatform/google-cloud-eclipse/eclipse/neon/gcp-eclipse-neon.target: Could not find "org.hamcrest.core/1.3.0.v201303031735" in the repositories of the current location -> [Help 1]
29084 [ERROR] 
29085 [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
29085 [ERROR] Re-run Maven using the -X switch to enable full debug logging.
29085 [ERROR] 
29085 [ERROR] For more information about the errors and possible solutions, please read the following articles:
29085 [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MavenExecutionException
```

By the way, why `eclipse/neon/.project` missing? I think it got lost accidentally.

--- a/eclipse/ide-target-platform/category.xml
+++ b/eclipse/ide-target-platform/category.xml
@@ -26,8 +26,8 @@
    <bundle id="org.eclipse.jetty.servlet" version="0.0.0"/>
    <bundle id="org.eclipse.jetty.server" version="0.0.0"/>
    <bundle id="org.eclipse.jetty.util" version="0.0.0"/>
-   <bundle id="org.hamcrest.core" version="1.3.0.v201303031735"/>
-   <bundle id="org.hamcrest.library" version="1.3.0.v201505072020"/>
+   <bundle id="org.hamcrest.core" version="1.3.0.v20180420-1519"/>
+   <bundle id="org.hamcrest.library" version="1.3.0.v20180524-2246"/>
    <bundle id="org.mortbay.jetty.server" version="6.1.26"/>
    <bundle id="org.mortbay.jetty.util" version="6.1.26"/>
    <bundle id="org.freemarker.freemarker" version="2.3.25.stable-incubating"/>

--- a/eclipse/neon/.project
+++ b/eclipse/neon/.project
@@ -1,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>gcp-eclipse-neon</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.m2e.core.maven2Builder</name>
+			<arguments>
+			</arguments>
+		</buildCommand>
+	</buildSpec>
+	<natures>
+		<nature>org.eclipse.m2e.core.maven2Nature</nature>
+	</natures>
+</projectDescription>

--- a/eclipse/neon/gcp-eclipse-neon.target
+++ b/eclipse/neon/gcp-eclipse-neon.target
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde?>
 <!-- generated with https://github.com/mbarbero/fr.obeo.releng.targetplatform -->
-<target name="GCP for Eclipse Neon" sequenceNumber="1520958315">
+<target name="GCP for Eclipse Neon" sequenceNumber="1527803874">
   <locations>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
       <unit id="org.eclipse.sdk.feature.group" version="4.6.3.v20170301-0400"/>
@@ -32,9 +32,9 @@
       <repository location="http://download.eclipse.org/webtools/repository/neon/"/>
     </location>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
-      <unit id="org.hamcrest.core" version="1.3.0.v201303031735"/>
+      <unit id="org.hamcrest.core" version="1.3.0.v20180420-1519"/>
       <unit id="org.hamcrest.integration" version="1.3.0.v201305210900"/>
-      <unit id="org.hamcrest.library" version="1.3.0.v201505072020"/>
+      <unit id="org.hamcrest.library" version="1.3.0.v20180524-2246"/>
       <unit id="org.slf4j.api" version="1.7.10.v20170428-1633"/>
       <unit id="org.slf4j.apis.jcl" version="1.7.10.v20160208-0839"/>
       <unit id="org.slf4j.apis.log4j" version="1.7.10.v20160208-0839"/>

--- a/eclipse/oxygen/gcp-eclipse-oxygen.target
+++ b/eclipse/oxygen/gcp-eclipse-oxygen.target
@@ -1,17 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde?>
 <!-- generated with https://github.com/mbarbero/fr.obeo.releng.targetplatform -->
-<target name="GCP for Eclipse Oxygen" sequenceNumber="1521647128">
+<target name="GCP for Eclipse Oxygen" sequenceNumber="1527803821">
   <locations>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
-      <unit id="org.eclipse.sdk.feature.group" version="4.7.3.v20180301-1154"/>
-      <unit id="org.eclipse.jdt.feature.group" version="3.13.3.v20180301-0715"/>
+      <unit id="org.eclipse.sdk.feature.group" version="4.7.3.v20180330-0919"/>
+      <unit id="org.eclipse.jdt.feature.group" version="3.13.4.v20180330-0919"/>
       <unit id="org.eclipse.m2e.feature.feature.group" version="1.8.3.20180227-2137"/>
       <unit id="org.eclipse.m2e.sdk.feature.feature.group" version="1.8.3.20180227-2137"/>
       <unit id="org.eclipse.m2e.wtp.feature.feature.group" version="1.3.3.20170823-1905"/>
       <unit id="org.eclipse.m2e.wtp.sdk.feature.feature.group" version="1.3.3.20170823-1905"/>
       <unit id="org.eclipse.mylyn.commons.feature.group" version="3.23.0.v20170503-0014"/>
-      <unit id="org.eclipse.jpt.jpa.feature.feature.group" version="3.5.100.v201705180510"/>
+      <unit id="org.eclipse.jpt.jpa.feature.feature.group" version="3.5.101.v201803161350"/>
       <unit id="org.eclipse.swtbot.eclipse.feature.group" version="2.6.0.201706141832"/>
       <unit id="org.eclipse.epp.logging.aeri.feature.feature.group" version="2.0.7.v20170906-1327"/>
       <unit id="org.eclipse.epp.logging.aeri.feature.source.feature.group" version="2.0.7.v20170906-1327"/>
@@ -32,9 +32,9 @@
       <repository location="http://download.eclipse.org/webtools/downloads/drops/R3.9.2/R-3.9.2-20171201000141/repository"/>
     </location>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
-      <unit id="org.hamcrest.core" version="1.3.0.v201303031735"/>
+      <unit id="org.hamcrest.core" version="1.3.0.v20180420-1519"/>
       <unit id="org.hamcrest.integration" version="1.3.0.v201305210900"/>
-      <unit id="org.hamcrest.library" version="1.3.0.v201505072020"/>
+      <unit id="org.hamcrest.library" version="1.3.0.v20180524-2246"/>
       <unit id="org.slf4j.api" version="1.7.10.v20170428-1633"/>
       <unit id="org.slf4j.apis.jcl" version="1.7.10.v20160208-0839"/>
       <unit id="org.slf4j.apis.log4j" version="1.7.10.v20160208-0839"/>

--- a/eclipse/photon/gcp-eclipse-photon.target
+++ b/eclipse/photon/gcp-eclipse-photon.target
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <?pde?>
 <!-- generated with https://github.com/mbarbero/fr.obeo.releng.targetplatform -->
-<target name="GCP for Eclipse Photon" sequenceNumber="1521657459">
+<target name="GCP for Eclipse Photon" sequenceNumber="1527803852">
   <locations>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
       <unit id="org.eclipse.sdk.feature.group" version="4.8.0.v20180308-0630"/>
@@ -32,9 +32,9 @@
       <repository location="http://download.eclipse.org/webtools/downloads/drops/R3.10.0/I-3.10.0-20180309052619/repository"/>
     </location>
     <location includeMode="planner" includeAllPlatforms="false" includeSource="true" includeConfigurePhase="false" type="InstallableUnit">
-      <unit id="org.hamcrest.core" version="1.3.0.v201303031735"/>
+      <unit id="org.hamcrest.core" version="1.3.0.v20180420-1519"/>
       <unit id="org.hamcrest.integration" version="1.3.0.v201305210900"/>
-      <unit id="org.hamcrest.library" version="1.3.0.v201505072020"/>
+      <unit id="org.hamcrest.library" version="1.3.0.v20180524-2246"/>
       <unit id="org.slf4j.api" version="1.7.10.v20170428-1633"/>
       <unit id="org.slf4j.apis.jcl" version="1.7.10.v20160208-0839"/>
       <unit id="org.slf4j.apis.log4j" version="1.7.10.v20160208-0839"/>

